pkg/util/util.go

```diff
@@ -109,10 +111,17 @@ func Forever(f func(), period time.Duration) {
 }
 
 // Until loops until stop channel is closed, running f every period.
+// Until is syntactic sugar on top of JitterUntil with zero jitter factor
+func Until(f func(), period time.Duration, stopCh <-chan struct{}) {
+	JitterUntil(f, period, 0.0, stopCh)
+}
+
+// JitterUntil loops until stop channel is closed, running f every period.
+// If jitterFactor is positive, the period is jittered before every run of f.
 // Catches any panics, and keeps going. f may not be invoked if
 // stop channel is already closed. Pass NeverStop to Until if you
 // don't want it stop.
-func Until(f func(), period time.Duration, stopCh <-chan struct{}) {
+func JitterUntil(f func(), period time.Duration, jitterFactor float64, stopCh <-chan struct{}) {
```

Comment:
util.Until would require jitter.JitterUntil and jitterUntil would require util.HandleCrash.

I am not against moving all the jitter logic (with Jitter itself too) to new jitter package. Atm, the wait.Jitter is used only on two place:
```vim
pkg/utils/wait.go:62:				adjusted = Jitter(duration, backoff.Jitter)
pkg/client/leaderelection/leaderelection.go:211:			time.Sleep(wait.Jitter(le.config.RetryPeriod, JitterFactor))
```

Still, there will be the import cycle. What about to move all the Until logic into wait package? I.e.:

* move util.Until, util.JitterUntil, util.Forever into wait
* move util.ForeverTestTimeout into wait

Then

* wait.Until would require util.HandleCrash
* wait.Forever and wait.JitterUntil would require wait.Until
* wait.ForeverTestTimeout not in util, thus no cyclic imports

However:
```vim
$ grep -rn "util.Until" | wc -l
57
$ grep -rn "Until" | wc -l
268
$ grep -rn "util.ForeverTestTimeout" | wc -l
60
$ grep -rn "ForeverTestTimeout" | wc -l
62
grep -rn "util.Forever" | wc -l
63
$ grep -rn "Forever(" | wc -l
6
```

So first step would be to copy all the relevant code from util to wait (otherwise it will be a lot of rebasing).
The second would be to start replacing all occurences of mentioned symbols.
Once it is finished write down some documetantation about "look util.Until is now wait.Until".

At the end:
```vim
pkg/util/wait: Until, Forever, ForeverTestTimeout, JitterUntil
```